extensions/wikia/CommunityPage/CommunityPageSpecialController.class.php

```diff
@@ -410,4 +452,20 @@ private function formatTotalEditorsNumber( $editors ) {
 
 		return $editors;
 	}
+
+	/**
+	 * @param $userId
+	 * @return string name of DS svg file with proper badge
+	 */
+	private function getUserBadge( $userId ) {
```

Comment:
As it's the first and only usage of badges in MW I'd leave it here - we also don't know if the pool of badges or priorities of them won't change between the usages. Then will be a good time to move it to a more reusable place.